src/searchForm/index.tsx

```diff
@@ -71,7 +75,6 @@ const StyledSearchForm = styled(SearchForm)`
     font-family: inherit;
     color: ${color.secondaryText};
     background: none;
-    display: flex;
     height: ${buttonHeight};
```

Comment:
This is messing with the vertical alignment in Firefox, having set to 100% fixes it (in desktop).
<img width="997" alt="Screen Shot 2020-03-26 at 13 31 03" src="https://user-images.githubusercontent.com/373381/77647812-62e96900-6f67-11ea-94f1-ab3eeda886b0.png">

Reply:
Ouch!

Reply:
Fixed by adding `align-items: center` so that the height is still correct for small devices